include/knowhere/config.h

```diff
@@ -564,7 +564,7 @@ class BaseConfig : public Config {
     }
 
     virtual Status
-    CheckAndAdjustForRangeSearch() {
+    CheckAndAdjustForRangeSearch(std::string* err_msg) {
```

Comment:
I didn't see anywhere you use this msg

Reply:
In scann `CheckAndAdjustForRangeSearch` will return err_msg if cpu not supported, similar with `CheckAndAdjustForSearch`